hardhat.config.js

```diff
@@ -47,17 +50,31 @@ module.exports = {
     },
     hardhat:{
       allowUnlimitedContractSize:false,
-      accounts:walletUtils.localWallet("1000000000000000000000",num=20),
+      gas: 6000000,
+      accounts:walletUtils.localWallet("1000000000000000000000000",num=20),
       forking : {
-        //url:`https://mainnet.infura.io/v3/${infuraKey}`
-        url:`https://eth-mainnet.alchemyapi.io/v2/${alchemyKey}`
+      //   url:`https://mainnet.infura.io/v3/${infuraKey}`
+      url:`https://eth-mainnet.alchemyapi.io/v2/${alchemyKey}`
       }
     },
     kovan:{
-      url:`https://kovan.infura.io/v3/${infuraKey}`,
+      //url:`https://kovan.infura.io/v3/${infuraKey}`,
+      url:"https://eth-kovan.alchemyapi.io/v2/DvW1I4OgMAVXJIw3zzfWHnQz1Lpeki9I",
```

Comment:
Don't upload infura API to github

Reply:
actually both are accepted if you keep keys in .infura and .alchemy so just removed the hard coded stuff